apps/communications/contacts/views/matching/js/matching_ui.js

```diff
@@ -70,7 +70,8 @@
                                           'suggestedDuplicatesMessage',
                                           params);
         } else {
-          document.title = this._('duplicatesFoundTitle');
+          navigator.mozL10n.setAttribute(document.title,
```

Comment:
it should be `setAttributes`. This is breaking the merging window. Did one of you try the patch on the device ? :/